precisionHelper.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * determines if precision is lost when parsing the JSON representation of a number to a javascript number
+ * @param {string} jsonNumber
+ * @param {number} javascriptNumber
+ * @return {boolean}
+ */
+function precisionWasLost (jsonNumber, javascriptNumber) {
+  return getSignificand(jsonNumber) !== getSignificand(JSON.stringify(javascriptNumber))
+}
+
+/**
+ * Takes in a string represention of a number and returns a simplified version of the significand
```

Comment:
How about this?

`Takes in a JSON represention of a number and returns its significand`

I think we should drop the next line of the comment, and have just a link to wikipedia, where it's explained more clearly.